dist/navitron.js

```diff
@@ -0,0 +1,275 @@
+                },
+                {
+                    display: 'block',
+                    easing: this.options.easing,
+                    duration: this.options.duration,
+                    mobileHA: this.options.mobileHA,
+                    begin: function() {
+                        // Setting z-index to make sure pane sliding in will always be on top
+                        // of pane that's being shifted out
+                        $targetMenu.css({
+                            zIndex: 2
+                        });
+                    },
+                    complete: function() {
+                        $targetMenu.attr('aria-hidden', 'false');
+                        $targetMenu.focus();
+                    }
+                }
+            );
+        },
+
+        shiftOut: function(currentPaneId, $button) {
+            if (currentPaneId === undefined) {
+                return;
+            }
+
+            var $shiftMenu = this._getTarget(currentPaneId);
+            var translateValue = this._getTranslateX($shiftMenu);
+
+            Velocity.animate(
+                $shiftMenu,
+                {
+                    translateX: [translateValue - this.options.shiftAmount + '%', translateValue + '%'],
+                    opacity: [this.options.fadeOpacityTo, 1]
+                },
+                {
+                    display: 'none',
+                    easing: this.options.easing,
+                    duration: this.options.duration,
+                    mobileHA: this.options.mobileHA,
+                    begin: function() {
+                        // Setting z-index to make sure pane shifting out will always be below
+                        // the pane that's being slided in
+                        $shiftMenu.css({
+                            zIndex: 1
+                        });
+                    },
+                    complete: function() {
+                        $shiftMenu.attr('aria-hidden', 'true');
+                        $button.attr('aria-expanded', 'true');
+                    }
+                }
+            );
+        },
+
+        slideOut: function(currentPaneId) {
+            if (currentPaneId === undefined) {
+                return;
+            }
+
+            var $currentMenu = this._getTarget(currentPaneId);
+
+            Velocity.animate(
+                $currentMenu,
+                {
+                    translateX: [0, '-100%']
+                },
+                {
+                    display: 'none',
+                    easing: this.options.easing,
+                    duration: this.options.duration,
+                    mobileHA: this.options.mobileHA,
+                    begin: function() {
+                        // Setting z-index to make sure pane shifting out will always be above
+                        // the pane that's being shifted back in
+                        $currentMenu.css({
+                            zIndex: 2
```

Comment:
Are you concerned about this hard set z-index? Should this be either relative to an existing element or has this worked fine for you?